code/modules/mob/living/carbon/human/species/station/station_vr.dm

```diff
@@ -317,13 +318,15 @@
 	min_age = 18
 	gluttonous = 0 //Moving this here so I don't have to fix this conflict every time polaris glances at station.dm
 	inherent_verbs = list(/mob/living/proc/shred_limb)
+	heat_discomfort_level = 294 //Prevents heat discomfort spam at 20c
```

Comment:
As it stands now, I've never had heat discomfort spam whilst a room is at 20C. I've only had this occur while playing a custom species with the Hot-Blooded trait